route/route.go

```diff
@@ -437,16 +437,7 @@ func (r *Router) requestToEvent(ctx context.Context, req *http.Request, reqBod [
 		return nil, err
 	}
 
-	return &types.Event{
-		Context:     ctx,
-		APIHost:     apiHost,
-		APIKey:      apiKey,
-		Dataset:     dataset,
-		Environment: environment,
-		SampleRate:  uint(sampleRate),
-		Timestamp:   eventTime,
-		Data:        data,
```

Comment:
Does that mean the sync.Pool also manages the `Data` field or it just manages the pointer to it?
Since the bulk of the data is in the `Data` field, it would be great if we can reuse the allocation for that

Reply:
As Event.Data is a map[string]interface{} (not a pointer), I think the pooled events will manage that too.

Reply:
Oh sorry, I meant that the map[string]interface{} is referenced by pointer not value. That means the `data` passed into the `type.NewEvent` method is a pointer to the map[string]interface{}. The creation of the allocation for map[string]interface{} is 3 lines above. I suspect that the `sync.Pool` is not managing the actual creation of the `data`